homeassistant/components/shelly/light.py

```diff
@@ -390,3 +395,39 @@ async def async_turn_on(self, **kwargs: Any) -> None:
     async def async_turn_off(self, **kwargs: Any) -> None:
         """Turn off light."""
         await self.call_rpc("Switch.Set", {"id": self._id, "on": False})
+
+
+class RpcShellyLight(ShellyRpcEntity, LightEntity):
+    """Entity that controls a light on RPC based Shelly devices."""
+
+    _attr_color_mode = ColorMode.BRIGHTNESS
+    _attr_supported_color_modes = {ColorMode.BRIGHTNESS}
+
+    def __init__(self, coordinator: ShellyRpcCoordinator, id_: int) -> None:
+        """Initialize light."""
+        super().__init__(coordinator, f"light:{id_}")
+        self._id = id_
+
+    @property
+    def is_on(self) -> bool:
+        """If light is on."""
+        return bool(self.coordinator.device.status[self.key]["output"])
+
+    @property
+    def brightness(self) -> int:
+        """Return the brightness of this light between 0..255."""
+        brightness_pct = self.coordinator.device.status[self.key]["brightness"]
+        return round(255 * brightness_pct / 100)
```

Comment:
This is duplicate code.  It would be nice to make it a function
```
light.py:        return round(255 * brightness_pct / 100)
light.py:        return round(255 * brightness_pct / 100)
```